ansible_runner/runner_config.py

```diff
@@ -734,11 +741,9 @@ def _parse_cli_execenv_cmd_playbook_args():
                 _ensure_path_safe_to_mount(host_path)
                 new_args.extend(["-v", "{}:{}".format(host_path, container_path)])
 
-        env_var_whitelist = ['PROJECT_UPDATE_ID']
-
         for k, v in self.env.items():
-            if k in env_var_whitelist:
-                new_args.extend(["-e", "{}={}".format(k, v)])
+            if k in self.env_var_whitelist:
+                new_args.extend(["-e", k])
```

Comment:
From a cursory reading, what happens to `v` here?

Reply:
If you pass `-e FOO` then docker / podman will pass the value of that environment on the host machine into the container. This is so that we can pass secret values without having them show up in `job_args` in the API, or in the process list on the computer. Credit goes to Shane here, otherwise I was struggling for any way to manage secrets in env vars.

Reply:
Gotcha, okay I get it now.